src/lib/mixpanel.ts

```diff
@@ -0,0 +1,11 @@
+export const MIXPANEL_TOKEN = process.env.NEXT_PUBLIC_MIXPANEL_TOKEN;
+
+export const mixpanelTrack = (eventName: string, props: any) => {
```

Comment:
```suggestion
export const mixpanelTrack = (eventName: string, ...props: unknown => {
```

인자를 여러개 받아올 수 있게 하면 어떨까요 !

> 단적으로 GA 이벤트와 믹스패널 이벤트를 한 번에 호출하는 유틸을 만들었을 때 사용될 수 있을 거 같아요

Reply:
> 

반영 완료 👍🏻 👍🏻